src/view/film-details.js

```diff
@@ -76,11 +83,11 @@ function createFilmDetailsTemplate(data, commentsFull) {
             </tr>
             <tr class="film-details__row">
               <td class="film-details__term">Writers</td>
-              <td class="film-details__cell">${writerts.length > 0 ? writerts.join(`, `) : writerts}</td>
+              <td class="film-details__cell">${writers.length > 1 ? writers.join(`, `) : writers}</td>
```

Comment:
выводить массив в разметку не круто. Сделай приведение к строке, хотя бы так:

```suggestion
              <td class="film-details__cell">${writers.length > 1 ? writers.join(`, `) : writers.join(``)}</td>
```